CMake/FindArmadillo.cmake

```diff
@@ -94,6 +94,20 @@ if(NOT _ARMA_USE_WRAPPER OR MSVC)
       endif()
     endif()
 
+    # On Linux, when statically linking against OpenBLAS, we must also manually
+    # link against -lgfortran and -lquadmath.  See
+    # https://gitlab.kitware.com/cmake/cmake/-/issues/23693 for more
+    # information.  When that issue is fixed, we may be able to remove this
+    # section.
+    if (NOT BUILD_SHARED_LIBS AND ${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+      string(TOLOWER "${LAPACK_LIBRARIES}" _lower_lapack_libs)
+      string(FIND "${_lower_lapack_libs}" "openblas" _openblas_found_index)
+      if (${_openblas_found_index} GREATER_EQUAL 0)
+	message(STATUS "Using static OpenBLAS on Linux; adding -lgfortran and -lquadmath...")
```

Comment:
Looking at the cross compilation test `aarch64-buildroot-linux-gnu/bin/ld: cannot find -lquadmath` this produces some issues.

Reply:
Ahh, let me be a bit more paranoid and disable this when cross-compiling.  4ed41f45f3